explainer.md

```diff
@@ -0,0 +1,66 @@
+# Capture all screens explainer
+
+## Introduction
+`getAllScreensMedia` is an API that allows clients to capture all monitors
+attached to a device at once without user interaction. It is only available in
+managed sessions for allow-listed web apps and it is a requirement
+that the user must be informed about capturing at all times. The usage
+indicator cannot be prevented by the web app.
+
+## Motivation
+Partners have asked for such an API in order to meet legal and internal
```

Comment:
A reader outside of CrOS would potentially be confused about who these partners are. I suggest employing a more neutral POV, speaking about how Web developers have certain needs.